### ISSUE TITLE
Geology attribute: private

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/arctos#2345]
With regard to https://github.com/ArctosDB/documentation-wiki/issues/133 I woke up last night thinking about the IPT and DwC out put for aggregators. Will the private (access) geology attribute also prevent those localities from appearing in the information shared to aggregators? We need to make sure it will.

**Comment by Jegelewicz** [comment]
Also, the specimens with the private (access) geology attribute map on the taxon name page and they shouldn't.

**Comment by dustymc** [comment]
I changed that from "yes, because implicit and sorta complicated reasons" to "yes, explicitly" and the DWC cache is rebuilding now. I'm not sure that's functionally different, but it'll make things a lot more obvious.

**Comment by Jegelewicz** [comment]
OK, we should probably test, but NMMNH paleo isn't being harvested yet. I can use a UTEP record to test. I'll post whatever one I change here and we can just wait a month or so until the next VertNet harvest to check it?

**Comment by dustymc** [comment]
>test

Don't let this discourage checking in other ways, but...

```
create table temp_iptcache as select * from ipt_tbl where organismID like 'http://arctos.database.museum/guid/NMMNH:Paleo:%';
```
[temp_iptcache.csv.zip](https://github.com/ArctosDB/arctos/files/3785703/temp_iptcache.csv.zip)


Also http://arctos.database.museum/guid/NMMNH:Paleo:4512 could have a much more specific identification - the part suggests it's not a plant or virus, geology suggests it's not an ammonite or gorila, ....

**Comment by Jegelewicz** [comment]
> Also http://arctos.database.museum/guid/NMMNH:Paleo:4512 could have a much more specific identification - the part suggests it's not a plant or virus, geology suggests it's not an ammonite or gorila, ....

Good call, I'll try to get "No ID, part=coprolite then ID=Animalia" into our migration protocol if I can.
